### PR TITLE
Improve events list appearance

### PR DESCRIPTION
**Pull request by justagoodguy-omega**

### CODE REVIEW

**Review by Ef55 — CHANGES_REQUESTED** [review]
LiNt WaRnInGs

**Comment by justagoodguy-omega** [comment]
> 
> 
> LiNt WaRnInGs

In this case it's

> 
> 
> LiNt WaRnInGs

In this case it's actually a feature not a bug, I just needed a view as padding for aesthetic purposes :)

**Comment by justagoodguy-omega** [comment]
> 
> 
> LiNt WaRnInGs

In this case it's actually a feature not a bug, I just needed a view as padding for aesthetic purposes :)

File: app/src/main/res/layout/day_events_list.xml [hunk 1]
```diff
@@ -3,14 +3,21 @@
...
         android:layout_width="fill_parent"
         android:layout_height="wrap_content"
-        android:divider="@color/teal_200"
-        android:dividerHeight="1dp" />
+        android:layout_marginHorizontal="5dp"
+        android:divider="@color/col_primary"
+        android:dividerHeight="3dp"
+        />
 
 </LinearLayout>
```
Comment (hugolan) [inline app/src/main/res/layout/day_events_list.xml:23]: warning here

**Review by hugolan — APPROVED** [review]
nothing much to say here just ui fixes

**Review by Ef55 — APPROVED** [review]

**Pull request merged**